suuchi-core/src/main/scala/in/ashwanthkumar/suuchi/store/Store.scala

```diff
@@ -8,4 +8,8 @@ trait WriteStore {
   def put(key: Array[Byte], value: Array[Byte]): Boolean
 }
 
-trait Store extends ReadStore with WriteStore
+trait DeleteStore {
```

Comment:
@brewkode Should we push this `remove` as part of `WritableStore`? Isn't it safe to assume that WriteStores also supports Deletes?